js_modules/dagster-ui/packages/ui-components/src/components/Button.tsx

```diff
@@ -261,6 +261,10 @@ export const JoinedButtons = styled.div`
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
   }
+
+  ${StyledButton}:focus-visible {
+    z-index: 1;
```

Comment:
Hmm I'm not sure a z-index without a position:relative will do anything unless the JoinedButtons happen to be a flex child. Doesn't seem harmful but maybe we need to throw a position: relative in there too?